client/options.go

```diff
@@ -41,16 +41,16 @@ func WithOpenCensusTracing() Option {
 // so that it will automatically inject trace-headers.
 //
 // Should be used when you trace your application with Datadog.
-func WithDatadogTracing(serviceName string) Option {
+func WithDatadogTracing(opts ...dd_http.RoundTripperOption) Option {
 	resourceNamer := func(req *http.Request) string {
 		return fmt.Sprintf("%s %s", req.Method, req.URL.String())
 	}
+	opts = append(opts, dd_http.RTWithResourceNamer(resourceNamer))
```

Comment:
should this check if there is already an ResourceNamer option?

Reply:
Thats not possible if we dont make our own options (which I prefer not to do).
I will update the PR to make it possible to set your own resourcenamer:
```
opts = append([]dd_trace.Option{dd_http.RTWithResourceNamer(resourceNamer)}, opts...)
```